sms_toolkit/messages/normalization.py

```diff
@@ -0,0 +1,22 @@
+from sms_toolkit.constants import unicode_to_gsm_map
+from sms_toolkit.utils import convert_to_unicode_characters
+
+
+def normalize_optimistic_gsm_characters(message, character_map=None):
+    """
+    Takes a message as input and returns a new message will all the appropriate unicode characters replaced with
```

Comment:
`will` -> `with`

Reply:
This sentence was badly worded thanks for the callout haha.